cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/preview/DefaultPreviewRunner.java

```diff
@@ -175,63 +175,86 @@ public void startPreview(PreviewRequest<?> previewRequest) throws Exception {
 
     final PreviewConfig previewConfig = previewRequest.getAppRequest().getPreview();
     ProgramController controller = programLifecycleService.start(
-      programId, previewConfig == null ? Collections.<String, String>emptyMap() : previewConfig.getRuntimeArgs(),
+      programId, previewConfig == null ? Collections.emptyMap() : previewConfig.getRuntimeArgs(),
       false);
```

Comment:
nit: more readable if this is now moved to the previous line

Reply:
Done